This file is part of MXE.
See doc/index.html for further information.

This patch has been taken from:
http://dev.exiv2.org/projects/exiv2/repository/revisions/2650

Index: trunk/src/jpgimage.cpp
===================================================================
--- trunk/src/jpgimage.cpp	(revision 2649)
+++ trunk/src/jpgimage.cpp	(revision 2650)
@@ -87,6 +87,7 @@ namespace Exiv2 {
 
     const char     Photoshop::ps3Id_[] = "Photoshop 3.0\0";
     const char*    Photoshop::irbId_[] = {"8BIM", "AgHg", "DCSR", "PHUT"};
+    const char     Photoshop::bimId_[] = "8BIM"; // deprecated
     const uint16_t Photoshop::iptc_    = 0x0404;
     const uint16_t Photoshop::preview_ = 0x040c;
 
Index: trunk/src/jpgimage.hpp
===================================================================
--- trunk/src/jpgimage.hpp	(revision 2649)
+++ trunk/src/jpgimage.hpp	(revision 2650)
@@ -65,6 +65,7 @@ namespace Exiv2 {
         // Todo: Public for now
         static const char     ps3Id_[]; //!< %Photoshop marker
         static const char*    irbId_[]; //!< %Photoshop IRB markers
+        static const char     bimId_[]; //!< %Photoshop IRB marker (deprecated)
         static const uint16_t iptc_;    //!< %Photoshop IPTC marker
         static const uint16_t preview_; //!< %Photoshop preview marker
 
